.gitignore

```diff
@@ -36,3 +36,4 @@ node_modules
 .vendor
 Goopfile.lock
 venv/
+.vscode/
```

Comment:
OK. So I put in in a separate commit, but in the same pull request?
Should I also reference the issue in this second commit?